var/spack/repos/builtin/packages/qemu/package.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2019 University of Colorado Boulder See the top-level
```

Comment:
The flake8 tests are failing because you changed the license header

Reply:
It was my understanding that copyright for contributions was retained by the contributor. Is that not the case, or must copyright be assigned to the spack project? If rights are retained, how is that indicated without violating this flake8 test?

Reply:
https://github.com/spack/contrib/blob/master/COPYRIGHT

> No copyright assignment is required to contribute to Spack.

Reply:
The only requirement of the flake8 tests is that the copyright comments at the top of the file remain unchanged. I don't know much else than that, so I'll defer to @tgamblin for how the legal stuff works.

Reply:
Any advice, @tgamblin?

Reply:
@anderbubble: we use a standard copyright header:

```
# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
# Spack Project Developers. See the top-level COPYRIGHT file for details.
```

The header is designed so that we don't have to change it every time a new contributor adds to a file (we have git history for that).  You (Boulder) are "other Spack Project Developers" and you do retain copyright (though note that you are putting this code under both Apache 2.0 and MIT licenses).

Other people (who may not be at UC Boulder) may later contribute to this file, so it may not end up being just yours.  I tried to just have the header just say "Spack Project Developers" but LLNL legal wanted explicit mention, since we started the project.

Given that, can you use the standard header?

Reply:
@tgamblin That makes sense enough to me. Sorry for all the back-and-forth (and the delay!)

I'll get the PR updated.